po/R-de.po

```diff
@@ -888,18 +895,27 @@ msgid ""
 "displayed. Therefore, the first and second degrees of freedom of both F-"
 "distributions switched places."
 msgstr ""
+"Wenn das Varianzverhältnis kleiner als 1 ist, wird der Kehrwert des "
+"Varianzverhältnisses angezeigt. Daher haben der erste und der zweite "
+"Freiheitsgrad bei beiden F-Verteilungen ihre Plätze getauscht."
 
 msgid ""
 "%1$sThe figure above shows two sampling distributions: the sampling "
 "distribution of the %1$s variance ratio when %2$s (left), and when %3$s%4$s "
 "(right)."
 msgstr ""
+"%1$sDie obige Abbildung zeigt zwei Stichprobenverteilungen: die "
+"Stichprobenverteilung des Varianzverhältnisses %1$s, wenn %2$s (links) und "
+"wenn %3$s%4$s (rechts)."
 
 msgid ""
 "On the other hand, if %3$s, the evidence would exceed the criterion  &mdash; "
 "and hence we would correctly claim that %4$s1 &mdash; at least %5$s%% of the "
 "time."
 msgstr ""
+"Andererseits, falls %3$s, würde die Evidenz das Kriterium überschreiten  "
+"&mdash; und daher würden wir korrekt behaupten, dass %4$s1 &mdash; in "
+"mindestens %5$s% der Fälle."
```

Comment:
Hi @JoKeyser thanks for your translation. please just note that don't change the placeholders(such `%s`, `%1$s` , `%2$i` things) when translate, for this case `%5$s%%` is a is a complete placeholder so you may not change it otherwise a error will in R/JASP. You can always keep an eye on this check, you can find out why this i18n-check failing [here](https://github.com/jasp-stats/jaspPower/actions/runs/5632052229/job/15259624098).

I also left a message for you on weblate.

cheers,
Shun

Reply:
Hi @shun2wang, thank you for pointing that out, this one slipped my attention. I'm generally aware of that, but I usually don't check the PRs, I will try to do that.

Thanks for fixing that one already!